.xe/specs/blueprint/spec.md

```diff
@@ -0,0 +1,413 @@
+- Enable parallel feature development within dependency tiers
+- Create living roadmap that tracks feature implementation status
+
+Explicit non-goals:
+
+- Detailed implementation plans for individual features (handled per-feature via start-rollout playbook)
+- Technology-specific decisions beyond architecture.md patterns
+- Migration strategies from predecessor systems
+
+## Scenario
+
+- As a **Product Manager**, I need to see the complete product roadmap so that I can prioritize features and communicate vision to stakeholders
+  - Outcome: Clear understanding of all features, their dependencies, and strategic phasing
+
+- As an **Architect**, I need to understand feature dependencies so that I can sequence implementation work correctly
+  - Outcome: No implementation order errors, ability to identify features that can be built in parallel
+
+- As an **Engineer**, I need complexity estimates and clear scope boundaries so that I can plan work and avoid scope creep
+  - Outcome: Accurate effort estimates, well-scoped features with single responsibilities
+
+- As a **Framework User**, I need to understand which features are available in each phase so that I can adopt Catalyst at the right maturity level for my needs
+  - Outcome: Clear phase assignments guide adoption timing
+
+## Success Criteria
+
+- All 27 features documented with unique IDs, names, descriptions, phase assignments, dependencies, complexity, and priority
+- Feature dependency graph is acyclic (no circular dependencies)
+- Features are organized into phase-relative tiers (1.1, 1.2, 2.1, etc.)
+- Phase 1 features have detailed specifications; Phase 2-5 features have high-level descriptions
+- Complexity distribution is balanced (not all Large features in one tier)
+- Every feature has clear 1-2 sentence scope description
+- Rollout plan tracks feature implementation status across all phases
+
+## Design principles
+
+- **Phase-driven prioritization**
+  > Features are assigned to phases based on strategic priorities from product.md (POC → Mainstream → Innovation → Platform → Enterprise → Scale). Phase assignments drive implementation sequencing and help users understand feature maturity.
+
+- **Dependency-first ordering**
+  > Within each phase, features are ordered by dependencies first (can't build X without Y), then by business value and risk. This prevents implementation order errors and enables parallel work on features in the same tier.
+
+- **Living specification**
+  > The blueprint is a living document that tracks feature implementation status and can be updated as product strategy evolves. Phase transition checkpoints allow re-blueprinting before starting new phases.
+
+- **Single responsibility per feature**
+  > Each feature represents a discrete, independently implementable capability with clear scope boundaries (1-2 sentences). Features can be extracted, replaced, or enhanced without affecting other features.
+
+## Requirements
```

Comment:
These requirements are wrong. They should be high-level requirements for the product, not requirements of the blueprint itself.